### ISSUE TITLE
[Discussion][Improvement]: Global switch for storage location of configurations

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#10100]
### Improvement description

followup to https://github.com/pimcore/pimcore/pull/9882

Right now, storage location can be defined via env-variables for each setting separately (e.g. `PIMCORE_WRITE_TARGET_IMAGE_THUMBNAILS=settings-store`). 
It might be useful, to have an additional global configuration (e.g. `PIMCORE_WRITE_TARGET=settings-store`) which can be overwritten with the specific ones. 

Usecases: 
- Don't 'forget' a setting in the configuration (also consider settings coming from bundles)
- Make sure, all configurations are disabled, also when a new setting is added with newer versions
- Bottom line: configure the env-variable once and have it setup for all configs - no need to care about in future and eventually change deployment chain, also when updating to new versions or installing additional bundles later on
